src/lib/destinations/webhook.ts

```diff
@@ -38,7 +38,7 @@ export const deliverToDestination = async (
   }
 
   return {
-    method,
+    method: method ?? "POST",
```

Comment:
@RossWilliams I was just looking this over post-merge to see what all has changed, and this caught my eye. This shouldn't be necessary because the zod type has a `.default` with this value. would be interested to know if something specific prompted this, or if you just wanted to make it explicit here